AsyncDisplayKit/ASTextNode+Subclasses.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <AsyncDisplayKit/ASTextNode.h>
+
+/**
+ * The subclass header _ASTextNode+Subclasses_ defines methods to be
```

Comment:
I appreciate the forethought in creating this file, but I actually find +Subclasses files hard to discover / work with unless it is a very complex class that is commonly subclassed (UIGestureRecognizer is well known for this, and also well documented, unlike ASTextNode subclassing).

Since we only have one method, let's put it in the regular ASTextNode interface towards the bottom, and declare prominently in the comments that it is provided as a subclass override hook.  If you can give an example of a use case (e.g. implementing a 3D projection effect or something, to provide additional backing store space), that would be illustrative to clients.